### PR TITLE
Fz12 ok 42

### PR DESCRIPTION
**Pull request by qbazielinski**
New data in user.json, weather.json, item-types.json and items.json. More or equal eight objects in all of json's components.

### CODE REVIEW

File: public/item-types.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
```
Comment (krichert) [inline public/item-types.json]: it should starts with [ and ends with ]
it is an array

File: public/items.json [hunk 2]
```diff
@@ -16,6 +16,72 @@
...
     "description": "Lorem ipsum",
     "proTip": "Lorem",
-    "img": "",
+    "img": "🕶️",
     "quantity": "default"
-}
\ No newline at end of file
+}
+
+{
```
Comment (krichert) [inline public/items.json]: same for this file -> [ ] at the begin and at the end

File: public/user.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
+{
```
Comment (krichert) [inline public/user.json]: we need only 1 user, so we need only 1 object

Comment (krichert) [reply public/user.json]: it will be represents data about actual logged user

File: public/weather.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (krichert) [inline public/weather.json]: again array [ ]

**Comment by krichert** [comment]
Just checked if json is correct by `npm start` and then in browser type http:localhost:3000/NAME_OF_YOUR_FILE.json 
you should see correctly all of your data

**Comment by qbazielinski** [comment]
Objects are removed, new arrays are done in files: item-types.json, items.json, user.json (just one user left), weather.json. All files was checked in the browser.

File: public/item-types.json [hunk 5]
```diff
@@ -0,0 +1,49 @@
+[
```
Comment (krichert) [inline public/item-types.json]: I mean array of objects:
[
 {
    "id": 2,
    "type": "clothes"
 },
 {
    "id": 3,
    "type": "cosmetics"
 },
 ...
]

Comment (krichert) [reply public/item-types.json]: and for others files the same :)

Comment (qbazielinski) [reply public/item-types.json]: Ok. I have changed it. Is it correct?

**Pull request closed without merging**